buds

```diff
@@ -37,8 +37,29 @@ log() {
         echo $(date +%Y%m%d-%H%M%S) : $* | tee -a $LOGFILE
 }
 
+
+
+# Check we have the tools we need to run the backup (rsync)
```

Comment:
I removed the test for tee on purpose - even a simple open-wrt router has tee in its repertoire, as a friend of mine pointed out.